config/sync/.htaccess

```diff
@@ -0,0 +1,23 @@
+# Deny all requests from Apache 2.4+.
```

Comment:
This branch is a merge between #159 and a branch that I was working on. I would like to hear @e0ipso judgement on this.